pkg/kubelet/kubelet.go

```diff
@@ -138,7 +138,7 @@ const (
 	housekeepingPeriod = time.Second * 2
 
 	// Period for performing eviction monitoring.
-	// TODO ensure this is in sync with internal cadvisor housekeeping.
+	// ensure this is kept in sync with internal cadvisor housekeeping.
```

Comment:
note: this is in sync with cadvisor housekeeping, so no action is required.

Reply:
actually, we now do eviction with on-demand metrics, so it doesn't matter how it relates to the housekeeping interval.